### PR TITLE
Jasmine tests on ci

### PR DESCRIPTION
**Pull request by adunkman**
@harvesthq/chosen-developers, Replaces #1462

Gets a very basic test harness in place for both jQuery and Prototype versions of Chosen... and they run on CI!

### CODE REVIEW

File: Gruntfile.coffee [hunk 1]
```diff
@@ -92,10 +94,44 @@ This file is generated by `grunt build`, do not edit it by hand.
...
+  grunt.loadNpmTasks 'grunt-contrib-concat'
+  grunt.loadNpmTasks 'grunt-contrib-watch'
+  grunt.loadNpmTasks 'grunt-contrib-copy'
+  grunt.loadNpmTasks 'grunt-contrib-clean'
+  grunt.loadNpmTasks 'grunt-contrib-compass'
+  grunt.loadNpmTasks 'grunt-contrib-cssmin'
+  grunt.loadNpmTasks 'grunt-build-gh-pages'
+  grunt.loadNpmTasks 'grunt-zip'
+  grunt.loadNpmTasks 'grunt-dom-munger'
+  grunt.loadNpmTasks 'grunt-contrib-jasmine'
```
Comment (pfiller) [inline Gruntfile.coffee]: These all look like bad merge stuff. #2207 made these task loads unnecessary!

File: Gruntfile.coffee [hunk 2]
```diff
@@ -92,10 +94,44 @@ This file is generated by `grunt build`, do not edit it by hand.
...
+
+  grunt.registerTask 'default', ['build']
+  grunt.registerTask 'build', ['coffee', 'compass', 'concat', 'uglify', 'cssmin']
```
Comment (pfiller) [inline Gruntfile.coffee]: The 2 above this are not necessary. I think they came from a bad merge.

File: Gruntfile.coffee [hunk 3]
```diff
@@ -92,10 +94,44 @@ This file is generated by `grunt build`, do not edit it by hand.
...
+  grunt.registerTask 'test',  ['coffee', 'jasmine']
+  grunt.registerTask 'gh_pages', ['copy:dist', 'build_gh_pages:gh_pages']
+  grunt.registerTask 'prep_release', ['build','zip:chosen','package_jquery']
```
Comment (pfiller) [inline Gruntfile.coffee]: The 2 above this are not necessary. I think they came from a bad merge.

File: Gruntfile.coffee [hunk 4]
```diff
@@ -92,10 +94,44 @@ This file is generated by `grunt build`, do not edit it by hand.
...
+  grunt.loadNpmTasks 'grunt-contrib-compass'
+  grunt.loadNpmTasks 'grunt-contrib-cssmin'
+  grunt.loadNpmTasks 'grunt-build-gh-pages'
+  grunt.loadNpmTasks 'grunt-zip'
+  grunt.loadNpmTasks 'grunt-dom-munger'
+  grunt.loadNpmTasks 'grunt-contrib-jasmine'
+
+  grunt.registerTask 'default', ['build']
+  grunt.registerTask 'build', ['coffee', 'compass', 'concat', 'uglify', 'cssmin']
+  grunt.registerTask 'test',  ['coffee', 'jasmine']
```
Comment (pfiller) [inline Gruntfile.coffee]: This is the only task that is needed in this PR

File: package.json [hunk 5]
```diff
@@ -18,10 +21,11 @@
     "grunt-contrib-compass": "~0.5.0",
     "grunt-contrib-concat": "~0.1.3",
     "grunt-contrib-cssmin": "~0.6.1",
-    "grunt-gh-pages": "~0.10.0",
```
Comment (pfiller) [inline package.json]: Can we move this back so it's not cluttering up the diff?

Comment (pfiller) [inline package.json]: nevermind - now they're in ABC order

**Comment by adunkman** [comment]
@pfiller thanks for the merge clean-up; couldn’t see this PR when it was opened yesterday since GitHub was all DDOS-y yesterday. 

Also thanks to @lenniboy for the original jasmines! :heart_decoration:

**Comment by pfiller** [comment]
:boom: :boom: :boom:

Thanks @adunkman and @lenniboy!

**Pull request merged**